plugin/fzf.vim

```diff
@@ -631,11 +631,6 @@ function! s:execute(dict, command, use_height, temps) abort
       call jobstart(cmd, fzf)
       return []
     endif
-  elseif has('win32unix') && $TERM !=# 'cygwin'
-    let shellscript = s:fzf_tempname()
-    call writefile([command], shellscript)
-    let command = 'cmd.exe /C '.fzf#shellescape('set "TERM=" & start /WAIT sh -c '.shellscript)
```

Comment:
`sh filename` should work here so `-c` is likely causing issues if `sh` cares about mingw/cygwin/msys2 file permissions. Read persmission is sufficient on Windows, unlike Unix.